test/integration/puppetlabs/services/master/environment_classes_int_test.clj

```diff
@@ -690,7 +690,7 @@
              _ (reset! wait-atom {:continue-promise continue-promise
                                   :wait-promise wait-promise})
              initial-response-future (future (get-env-classes "production"))]
-         (is (true? (deref wait-promise 10000 :timed-out))
+         (is (true? (deref wait-promise 20000 :timed-out))
```

Comment:
Yes, it does! (or it did on my computer at least). I meant to remove this before I pushed it up, sorry!